features/step_definitions/pod.rb

```diff
@@ -249,6 +249,18 @@
   @result  = res_cache
 end
 
+Given /^I collect the deployment log for pod "(.+)" until it becomes :([^\s]*?)$/ do |name, status|
+  opts = {resource_name: name}
+  timeout = timeout ? Integer(timeout) : 15 * 60   # just put a timeout so we don't hang there indefintely
```

Comment:
Perhaps `timeout = 15 * 60` as we do not define timeout in the step name.

Reply:
@zhouying7780 could you please check and update this, so that lxia could merge the patch as it is causing one of the test to fail ? Thanks !!